walkgen_params:
  params:
    N_phase_return: 2 # Number of step to return (N_phase_return surfaces for each foot)

  gait:
    type: "trot" # Only "walk" or "trot" working
    dt: 0.01
    horizon: 125 # Horizon length of the queue of contact.
    nsteps: 1 # Number of iteration.
    stepHeight: 0.15 # Step height [m]
    trot:
      N_ds: 80
      N_ss: 35
      N_uds: 0
      N_uss: 0
    walk:
      N_ds: 90
      N_ss: 70
      N_uds: 0
      N_uss: 0
